pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+env =
+    FLASK_ENV=unit_test
+    FLASK_DEBUG=1
+    D:DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/postgres
```

Comment:
Could the test DB use a different database name than the suggested app one? Else there is a risk of them sharing a database locally and the database being cross-contaminated.

Reply:
Here is an example from Digital Marketplace tearing up a separate test DB: https://github.com/Crown-Commercial-Service/digitalmarketplace-api/blob/af031df96f82a062bd92c589efb1beca0dfed3b0/tasks.py#L23-L26